Fixed css

hassaanahmedc opened the pull request:
I did styling of this project and made it responsive for multiple screens.
also i tweaked some HTML code like added label tags for input names to style the content and wrapped the whole content inside a 'Wrapper' div.
It was amazing working on the project. 👍

hassaanahmedc committed:
Countdown Timer Mobile Verison Done

hassaanahmedc committed:
Desktop version done

sanskar-mk2 commented:
Looks good. Here are few changes that can be done:
- From wrapper, remove the overflow since it adds non functional scroll in browser.
- Don't have absolute position since things get hidden as more timers appear.
  - Wrap wrapper inside another div and make it take whole screen and then you can flex align the wrapper to center
- The calendar and time icon is hard to see because of the color. Adjust the colors.
- It would look nice if all the timers have borders and background colors.

These are my suggestions. 
Good work.

hassaanahmedc committed:
Additional Changes

hassaanahmedc committed:
Additional Changes (fixed height of container)

hassaanahmedc commented:
Thank you for the ideas to help me improve the design. 
I made the following changes according to your instructions.

- Removed overflow x and y properties.
- Replaced position absolute with flexbox.
- Fixed Height of the container.
- Fixed color of calender and clock icons to make them visible.
- Increased the font size of countdown time and added border-bottom to make it look more appealing.

It was alot of fun working on the project , learned and experienced alot of new things which will definitely help me to improve as a developer.

sanskar-mk2 commented:
Thank you. Merged.
